x/auth/posthandler/tips.go

```diff
@@ -47,5 +49,10 @@ func (d tipDecorator) transferTip(ctx sdk.Context, sdkTx sdk.Tx) error {
 		return err
 	}
 
-	return d.bankKeeper.SendCoins(ctx, tipper, tipTx.FeePayer(), tipTx.GetTip().Amount)
+	coins := tipTx.GetTip().Amount
+	if err := d.bankKeeper.IsSendEnabledCoins(ctx, coins...); err != nil {
+		return fmt.Errorf("cannot tip these coins: %w", err)
+	}
```

Comment:
Q: Why not move the `IsSendEnabledCoins` check directly into `SendCoins` and avoid having to have this check elsewhere potentially missing it.

Reply:
This made sense to me, however we don't know how people are using our API and expect them to work.
Do you think that behavior change on such a core feature is acceptable?

Reply:
I would imagine if a coin cannot be sent, as dictated by `SendEnabled`, then I'd expect `SendCoins` to fail/error.

Reply:
Yes, I agree. However, I would not assume that for all modules.
I would advocate doing it (add `IsSendEnabledCoins` in `SendCoins`) for v0.48 and v0.47 but not for v0.46.
What do you think?

Reply:
Yes, I agree. However, we don't know if module devs made another (even though wrong) assumption.
Say you have a module that allows an authority only to moves send disabled denom, then this change in the SDK would break it.

Reply:
I would say that breaks the safety model then and introduces a misuse of the APIs. If such a case exists, then I would introduce a new `SendCoinsUnsafe` API or something similar that bypasses such checks.

Reply:
I am more afraid to break, people's code by a change of code behavior.
I've looked deeper when is used `IsSendEnabledCoins`, and it is usually used way before calling `SendCoins` and before creating accounts in vesting, or in bank send in the SDK.
`SendCoins` on the other hand, is very, very much used and very commonly without that check: https://cs.github.com/?scopeName=All+repos&scope=&q=.SendCoins%28+language%3AGo.

I am going to merge it now, but this is maybe something we need to talk about. Personally, I agree with you, `SendCoinsFromModuleToAccount` checks if it is a blocked account, for instance, so we do have safety checks in another bank send method.

Reply:
SGTM